src/sidebar/views/overview.ts

```diff
@@ -0,0 +1,138 @@
+import { format } from 'node:path';
+import { Command, Event, EventEmitter, ExtensionContext, TreeDataProvider, TreeItem, TreeView, window } from 'vscode';
+import { ExtensionApi } from '../../backend/api';
+import { CheckerMetadata } from '../../backend/types';
+
+export class OverviewItem {
+    constructor(private label: string | (() => string), private command?: Command) {}
+
+    getTreeItem(): TreeItem | Promise<TreeItem> {
+        const label = typeof this.label === 'string' ? this.label : this.label();
+
+        const node = new TreeItem(label);
+        node.command = this.command;
+        node.description = this.command?.tooltip;
+
+        return node;
+    }
+}
+
+export class OverviewView implements TreeDataProvider<string> {
+    private tree?: TreeView<string>;
+
+    // FIXME: Export the keys into an enum, or otherwise check in the itemsList if an entry exists
+    private items: {[id: string]: OverviewItem} = {
+        'loading': new OverviewItem('Loading overview, please wait...'),
+
+        'notfound': new OverviewItem('CodeChecker run not found.'),
+        'notfound2': new OverviewItem('Run CodeChecker, reload metadata,'),
+        'notfound3': new OverviewItem('or set the output folder to get started'),
+
+        'files': new OverviewItem(() => `Files analyzed: ${ExtensionApi.metadata.sourceFiles.size}`),
+        'lastRun': new OverviewItem(() => `Last run on: ${
+            new Date(ExtensionApi.metadata.metadata!.timestamps.begin /* seconds */ * 1000).toLocaleString()
+        }`),
+        'lastRunFiles': new OverviewItem(() => `Last run's files: ${ExtensionApi.metadata.metadata!.action_num}`),
```

Comment:
I think this is a redundant information because above we will print the number of analyzed translation units.
```suggestion
```

Reply:
There's a difference, the `files` shows the total number of analyzed files, while this one only shows the last run's files.
But this is a meaningless statistic anyways, so removed.